### ISSUE TITLE
search results item metadata display issues

### ISSUE THREAD

**Issue by jduss4** [CDRH/orchid#102]
Needs for various projects:

- separate multivalued fields (creators) in display with semi-colon or other character
- sort multivalued fields (creators.name) so that when specific sort is selected it will look more logical

Example:  sorting on "creator name descending" currently might get these results:
`Fowler, Alice; Whitman, Walt`
`Whitman, Walt`
`Abrams, C.P. ; Wanda, Willa`
The sorting actually happening is Whitman, Whitman, Wanda, but the display looks like it's out of order.

**Comment by jduss4** [comment]
Not sure if we should also make sure that if the sort of a field (let's keep talking creator.name since that's actually my problem at the moment) if being sorted descending.....if therefore the display of the metadata should ALSO be in reverse?  or if we should always just sort the creators ascending in the metadata...

**Comment by techgique** [comment]
Clarification question:
Did you mean the sorting happening is `Fowler…, Whitman…, Wanda…`? Sorting on the beginning of each semi-colon-separated string?

**Comment by jduss4** [comment]
Right, so with the API fix which allows sorting in multivalued fields, you can choose whether a sort should be using the minimum value or the maximum value in the the array of fields.  When a user clicks "sort ascending by creator" the multivalued search is carried out by using the minimum value, so that it picks "A" instead of "Z" for ascending.  Vice versa for a descending search.

However, though the search results being returned should be accurate, the UI doesn't currently present the multivalued field in a particular manner, so for the user it looks confusing when they searched for "ascending" but the top result might be "Whitman" at the beginning of the creators line, even though it was actually returning a result for "Balderdash" which just happened to be displayed second.  I think it makes a lot of sense to figure on sorting the multivalued fields when that particular field was relevant to the selected sort.

**Comment by techgique** [comment]
Thanks for the explanation. I agree, it seems like we should add the flexibility to make certain sorts behave the way we determine more useful or people will probably expect.
